validator/etc/txnvalidator.js.example

```diff
@@ -1,67 +0,0 @@
-{
```

Comment:
what replaces txnvalidator.js.example?  If multiple files (which I'm wary of), how does a tester or deployer determine what files need to be modified and where to find them all?